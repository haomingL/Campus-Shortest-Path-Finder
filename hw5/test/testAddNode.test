# Create a Graph called graph1
CreateGraph graph1

# Add a pair of Nodes 
AddNode graph1 n1
AddNode graph1 n2

# Add an edge
AddEdge graph1 n1 n2 e1

# Print the nodes in the graph and the outgoing edges from n1
ListNodes graph1
ListChildren graph1 n1

# Create a Graph called graph2
CreateGraph graph2

# add some Nodes
AddNode graph2 n1
AddNode graph2 n2
AddNode graph2 n3
AddNode graph2 n4
AddNode graph2 n5

# Add some edges
AddEdge graph2 n1 n2 e12
AddEdge graph2 n1 n3 e13
AddEdge graph2 n1 n4 e14
AddEdge graph2 n2 n3 e23
AddEdge graph2 n3 n5 e35
AddEdge graph2 n2 n4 e24
AddEdge graph2 n4 n5 e45

# Print the nodes in the graph and the outgoing edges from n1, n2 ...
ListNodes graph2
ListChildren graph2 n1
ListChildren graph2 n2
ListChildren graph2 n3
ListChildren graph2 n4