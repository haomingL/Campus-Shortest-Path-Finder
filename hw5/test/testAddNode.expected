# Create a Graph called graph1
created graph graph1

# Add a pair of Nodes 
added node n1 to graph1
added node n2 to graph1

# Add an edge
added edge e1 from n1 to n2 in graph1

# Print the nodes in the graph and the outgoing edges from n1
graph1 contains: n1 n2
the children of n1 in graph1 are: n2(e1)

# Create a Graph called graph2
created graph graph2

# add some Nodes
added node n1 to graph2
added node n2 to graph2
added node n3 to graph2
added node n4 to graph2
added node n5 to graph2

# Add some edges
added edge e12 from n1 to n2 in graph2
added edge e13 from n1 to n3 in graph2
added edge e14 from n1 to n4 in graph2
added edge e23 from n2 to n3 in graph2
added edge e35 from n3 to n5 in graph2
added edge e24 from n2 to n4 in graph2
added edge e45 from n4 to n5 in graph2

# Print the nodes in the graph and the outgoing edges from n1, n2 ...
graph2 contains: n1 n2 n3 n4 n5
the children of n1 in graph2 are: n2(e12) n3(e13) n4(e14)
the children of n2 in graph2 are: n3(e23) n4(e24)
the children of n3 in graph2 are: n5(e35)
the children of n4 in graph2 are: n5(e45)